code/game/gamemodes/antag_datum.dm

```diff
@@ -0,0 +1,197 @@
+/mob/living
+	var/list/antag_datums
```

Comment:
I'm pretty sure this will need to be a mob or mind var because blob. Thanks blob.